src/main.js

```diff
@@ -0,0 +1,51 @@
+import { createTripInfoTemplate } from './view/info-view.js';
+import { createNavigationTemplate } from './view/navigation-view.js';
+import { createFiltersTemplate } from './view/filters-view.js';
+import { createSortingTemplate } from './view/sorting-view.js';
+import { createListPointsTemplate } from './view/list-points-view.js';
+import { createEditingPointTemplate } from './view/point-edit-view.js';
+import { createNewPointTemplate } from './view/point-create-view.js';
+import { createShowPointTemplate } from './view/point-view.js';
+import { createStatisticsTemplate } from './view/stats-view.js';
+import { createMessagesTemplate } from './view/messages-view.js';
+
+const POINT_COUNT = 3;
+
+const RenderingLocation = {
+  BEFOREBEGIN: 'beforebegin',
+  AFTERBEGIN: 'afterbegin',
+  BEFOREEND: 'beforeend',
+  AFTEREND: 'afterend'
+};
```

Comment:
```suggestion
const RenderingLocation = {
  BEFORE_BEGIN: 'beforebegin',
  AFTER_BEGIN: 'afterbegin',
  BEFORE_END: 'beforeend',
  AFTER_END: 'afterend'
};
```

Именование констант принято разделять через _